Code/webapp/app.js

```diff
@@ -4,5 +4,6 @@ angular.module('mainApp', [
     'vip-projects',		//Features Module
 	'ProjectProposalController',
 	'routes',
-	'projectApplicationController'
+	'userRegistrationController',
```

Comment:
dont think you merged properly, this 'projectApplicationController' dependency was removed